playbooks/frontend.yml

```diff
@@ -150,3 +150,8 @@
         name: '*'
         security: yes
         state: latest
+
+    - name: Ensure Anacron is disabled by making the binary non executable
```

Comment:
Does it make more sense to use:

```
service:
  name: anacron
  state: stopped
```

????

Reply:
This is not a running service that shows up.
Uninstalling anacron will also remove the cronie dependency which is required by crond.
Making it non executable seems to be the best way to achieve this for now.